src/main.py

```diff
@@ -174,16 +174,16 @@ def build_images(args):
                                                     args.force, args.skip_tests)
     generate_release_notes(target_version)
 
+    # Upload to ECR before running tests so that only the exact image which we tested goes to public
```

Comment:
Should we add a TODO to revisit this after our tests are stabilized?

Reply:
Sure added